src/datasets/argument_relation_detection_dataset.py

```diff
@@ -13,28 +13,34 @@ def __init__(
         self,
         json_path: Path,
         tokenizer: PreTrainedTokenizer,
+        test_mode: bool = False,
     ):
         with open(json_path, "r") as f:
             data = json.load(f)
-        self.texts: list[tuple[str, str]] = [(sample[0], sample[1]) for sample in data]
-        self.labels: list[int] = [sample[2] for sample in data]
+        self.texts: list[tuple[str, str]] = [(sample[1], sample[2]) for sample in data]
```

Comment:
We can't just change these numbers here, the code is already used for training in task 2, and we don't want it to stop working. 

There are a couple of ways to do this, the simplest is probably to have an additional argument in the constructor:

task: str = "task2"

then here:
if task == "task1":
    self.texts == something
elif task == "task2":
    self.texts = something else
else:
    raise ValueError(some description)

and add a short comment explaining that the datasets are in different formats.